pkg/plugins/platform/local_test.go

```diff
@@ -62,13 +68,56 @@ func TestMain(m *testing.M) {
 		if err != nil {
 			log.Fatalf("Prepare() failed on TestSite.Prepare(), err=%v", err)
 		}
+
+		switchDir := TestSites[i].Chdir()
+		runTime := testcommon.TimeTrack(time.Now(), fmt.Sprintf("%s Start", TestSites[i].Name))
+
+		testcommon.ClearDockerEnv()
+
+		app, err := platform.GetPluginApp("local")
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		err = app.Init(TestSites[i].Dir)
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		err = app.Start()
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		runTime()
+		switchDir()
 	}
 
 	log.Debugln("Running tests.")
```

Comment:
Sadly, `-short` is not available in TestMain, or I would have done the sizing there in the first place :(

We might be able to parse go test args ourselves, but that feels really really ugly.